go.sum

```diff
@@ -151,6 +151,8 @@ github.com/openshift-online/ocm-sdk-go v0.1.128 h1:AlX7PkZOuZJPBzMM2CgF22YGMp6M9
 github.com/openshift-online/ocm-sdk-go v0.1.128/go.mod h1:3i3a/LEYtC7DMor3N94PTPn1+0qq+Fk+iLjt1Z0WVCs=
 github.com/openshift-online/ocm-sdk-go v0.1.129 h1:LjqS4OfpTspoDk6+R8IqoqSt4DJl7hhSSHkZswlXSpk=
 github.com/openshift-online/ocm-sdk-go v0.1.129/go.mod h1:3i3a/LEYtC7DMor3N94PTPn1+0qq+Fk+iLjt1Z0WVCs=
+github.com/openshift-online/ocm-sdk-go v0.1.131 h1:hOa26HGY0/OY3pLg/8IqdfcU9o6rSZKc5zxywnmwUDY=
+github.com/openshift-online/ocm-sdk-go v0.1.131/go.mod h1:3i3a/LEYtC7DMor3N94PTPn1+0qq+Fk+iLjt1Z0WVCs=
```

Comment:
You should run `go mod tidy` to remove old versions from this file

Reply:
thanks, done